jenkins-ansible/README.md

```diff
@@ -0,0 +1,194 @@
+# 
+# PetPortal Ansible-Jenkins
+
+This is my Pet Portal which uses Ansible to provision the targeted environemnt and Jenkins to Build and Test the
+ applicaion.  This blueprint also leverages the xlr-xlrelease-plugin to dynamically tag the releases with the 
+ Service Now tickets.
+
+## Setup Steps
+1. Clone this repository
+2. Configure your ~/.xebialabs/config.yaml for your connection to the XL platform tools
+```
+    xl-deploy:
+      username: admin
+      password: admin
+      url: http://localhost:4516/
+    xl-release:
+      username: admin
+      password: admin
+      url: http://localhost:5516/
+    blueprint-repository:
+      username: ""
+      password: ""
+      # file:// protocol is not supportted so you'll need to run a local http server
+      # E.g. in blueprint directory run: python -m SimpleHTTPServer 3516
+      url: ttp://localhost:3516
+```
+_3. Create the Blueprints
+
+```
+| => xl --config config.yaml blueprint  
+? Choose a blueprint: jenkins-ansible  
+? What is the XL Release folder name? PetPortal Project  
+? What is the Ansible Host name? ansible  
+? What is the Jenkins server name? Jenkins  
+? Confirm to generate blueprint files? (Y/n)   
+```
+
+_4. Create a docker-compose.yaml file **from contents at the bottom of this page** and run it to create a demo environment
+```
+| => docker-compose up --build
+```
+
+_5. Apply the blueprint to the XL platform, in this case it will create a XL Release template to demonstrate the pipeline
+```
+| => xl --config ./config.yaml apply -f ./xebialabs/xlr-pipeline.yaml  
+```
+_6. Navigate to XL Release and start a release from the new template  
+![XLReleaseTemplate](images/releaseTemplate.png)  
+![StartRelease](images/startRelease.png)
+
+
+## Docker Compose
+
+Use this docker-compose.yaml file to create a demo environemmt to test this blueprint
+### docker-compose.yaml
```

Comment:
We should move the docker setup to another repo or you can also just generate the docker-compose file required as part of the blueprint

Reply:
Not sure what you mean by doing it as part of the blueprint. I will just say that you must have a jenkins host and Ansible host defined. And I can point to the demo repo docker image as an example.

Reply:
I meant to generate the `docker/docker-compose.yaml` file when someone runs the blueprint